Xamarin.Forms.ControlGallery.iOS/GalleryPages/RegionalLocale.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using Foundation;
+using Xamarin.Forms;
+using Xamarin.Forms.Controls.GalleryPages.DateTimePickerGalleries;
+
+[assembly: Dependency(typeof(Xamarin.Forms.ControlGallery.iOS.GalleryPages.RegionalLocale))]
+namespace Xamarin.Forms.ControlGallery.iOS.GalleryPages
+{
+	public class RegionalLocale : ILocalize
```

Comment:
Out of curiosity is this just providing an extension point for people and this is just your test for that?

Is this class here useful outside of the gallery in any way?

Reply:
It's not useful outside the gallery. I think we just wanted the users to be able to tell the device locale and language from the control gallery page that I'm using. However, on iOS getting the device culture makes the language match the region. So, even if the region is US and language is French, the device culture would give fr-FR rather than en-FR which is why we had to create this class.